cmd/restic/cmd_backup.go

```diff
@@ -203,12 +202,55 @@ func readLinesFromFile(filename string) ([]string, error) {
 	return lines, nil
 }
 
+// readFilenamesFromFileRaw reads a list of filenames from the given file,
+// or stdin if filename is "-". Each filename is terminated by a zero byte,
+// which is stripped off.
+func readFilenamesFromFileRaw(filename string) (names []string, err error) {
```

Comment:
I suggest we rename this (and the corresponding one below) to `readPathsFromFileRaw`, as it's really paths we're reading, not just filenames.

Reply:
The convention in the Go stdlib is to call paths filenames (or just names) as well. See the os pkg docs, for example.

Reply:
I see, fair enough!